jstests/ssl/x509/mkcert.py

```diff
@@ -520,17 +571,42 @@ def process_ecdsa_leaf(cert):
     reqargs = ['openssl', 'req', '-new', '-key', key, '-out', csr, '-subj', subject]
     x509args = ['openssl', 'x509', '-in', csr, '-out', pem, '-req', '-CA', ca, '-CAkey', ca, '-days', '7300', '-sha256', '-set_serial', serial]
     if mode == 'server':
-        reqargs.append('-reqexts=v3_req')
+        reqargs = reqargs + ['-reqexts', 'v3_req']
         extfile = tempfile.mkstemp()[1]
-        open(extfile, 'wt').write('subjectAltName=DNS:localhost,DNS:127.0.0.1')
+        open(extfile, 'wt').write('basicConstraints=CA:FALSE\n')
```

Comment:
This template is getting big enough we might *consider* putting it in an external file which we load from here.

Reply:
Noted! I've left it the way it is for now since other drivers work is more pressing at the moment, but if you feel strongly about making this optimization now as part of this PR, I can look into it